Manage treatment without previous biopsy

mickohanlon23 opened the pull request:
This PR Resolves [CCSMCDS-7](https://jira.mitre.org/browse/CCSMCDS-7). This removes the requirement for a high grade biopsy or cytology result to be present before a cervical treatment in order for a patient to be managed by Rare Abnormality 4 (Section J) or Risk Table 5. This functionality is parameterized so that the requirement for a high grade test before a treatment can be turned off/on. A test case is added to show that a patient is able to enter Risk Table 5 without a CIN2 or CIN 3 biopsy before their treatment. More test cases are added to show that each of the Section J Rare Abnormality 4 recommendations can be reached without high grade test results before the cervical precancer treatment.

mrnosal reviewed (approved):
Reviewed. Built and ran tests.

mickohanlon23 commented:
- Merged in main branch
- Updated ELM file and encoded ELM in FHIR Libraries
- Updated recommendation text in newly added test cases to align with recent recommendation text update.
